config/samples/flows_v1alpha1_flowcollector.yaml

```diff
@@ -94,7 +94,6 @@ spec:
     imagePullPolicy: IfNotPresent
     port: 9001
     logLevel: info
-    autoscaler: null
```

Comment:
could you add a sample autoscaler config, but commented out? I find it quite convenient when trying configs that are not obvious

Reply:
Done!

Reply:
thanks! Actually better than commented out :)